### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by HermanVulkers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- React
+- NextJS
+- Styled components
+- NPM
+- ESLint
+- Prettier
+
+## Authors and acknowledgment
+
+Many thanks to the other trainees and all the coaches for the help they gave me during this project.
```
Comment (fabianovangool) [inline README.md:27]: ✨

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Example
+
+![Screenshot Team Awesome](../team-awesome-herman/public/assets/images/screenshot.png)
+
+## Usage
+
+The two main features on Team Awesome are the ability to add time entries on the landing page (Timesheets) and new team members on the Team members page. Press the green button in the top right of the subheader to add a new entry or a team member. The new item will render on the main page, behind the dialog. Click the cancel button or the close icon (top right) to close the dialog. On the time entries page, entries can be filtered by client, using a dropdown menu located top right, above the entries. Time entries can also be removed by clicking the bin icon. On the team members page, members can be sorted by different values, such as name or starting date.
+
+## Usage
```
Comment (fabianovangool) [inline README.md:13]: I think this heading is supposed to be called Technologies

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+The follow technologies were used:
+
+- TypeScript
+- React
+- NextJS
+- Styled components
+- NPM
+- ESLint
+- Prettier
+
```
Comment (fabianovangool) [inline README.md:24]: I think there's a bit missing about how to run the app (see project ticket req's)

**Review by fabianovangool — CHANGES_REQUESTED** [review]

**Review by fabianovangool — APPROVED** [review]

**Review by hans-acc — APPROVED** [review]
perfection 💯

**Review by fabianovangool — APPROVED** [review]

**Pull request merged**